src/Salt/UserBundle/Controller/UserController.php

```diff
@@ -81,6 +81,16 @@ public function newAction(Request $request)
             $encryptedPassword = $this->get('security.password_encoder')
                 ->encodePassword($targetUser, $targetUser->getPlainPassword());
 
+            // email the new user
+            $email = $targetUser->getUsername();
+            $message = (new \Swift_Message('Hello Email'))
+            ->setFrom('send@example.com')
+            ->setTo($email)
+            ->setSubject('Your account has been created')
+            ->setBody('Thank you! Your account has been created and you will be contacted in 2 business days when it is active.');
+
+            $this->get('mailer')->send($message);
```

Comment:
The message should not be sent until after the command succeeds (in case creating the user fails).

We should not send an email if no mail service is configured.
Sending the email should be an optional feature that can be enabled/disabled.

Reply:
Moved the code after the user creation code in the try exception.